client/state/account-recovery/reset/actions.js

```diff
@@ -3,9 +3,12 @@
  */
 import wpcom from 'lib/wp';
 import {
-	ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
-	ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
 	ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
+	ACCOUNT_RECOVERY_RESET_REQUEST_RESET,
+	ACCOUNT_RECOVERY_RESET_REQUEST_RESET_SUCCESS,
+	ACCOUNT_RECOVERY_RESET_REQUEST_RESET_ERROR,
```

Comment:
Ah, totally. I have no idea why they were there in the first place 😅
Fixed by 6d6d057